.travis.yml

```diff
@@ -3,6 +3,7 @@ language: julia
 os:
   - linux
   - osx
+  - windows
```

Comment:
I'm not sure whether Windows in Travis is well supported. Most developers are still using Appveyor.

Reply:
That is the result of the post that I saw here: https://discourse.julialang.org/t/julia-is-now-available-on-travis-windows-builds/27187